content/ja/handbook/engineering/development/sec/delineate-sec.md

```
@@ -0,0 +1,141 @@
+<p class="my-3 text-sm text-gray-600 italic">DevOps ステージ図 (Application Security Testing · Security Risk Management)は <a href="https://handbook.gitlab.com" rel="external noopener">原文 (英語)</a> を参照してください。</p>
+
+
+## このページが存在する理由
+
+明確でない場合に備え、各機能セットに [DRI](/handbook/people-group/directly-responsible-individuals/) を設けるという精神のもと、このページの目的は、どのエンジニアリンググループが製品のどの部分と特定の意思決定に責任を持つかを明示的に定義することです。
+
+## ページ/機能の責任
+
+| ページ/機能 | PM | 主担当グループ | 例 |
+| ---      | ---      | ---      | ---      |
+| [Secure パートナーオンボーディングドキュメント](https://docs.gitlab.com/ee/development/integrations/secure_partner_integration.html) | 
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 4; Actual: 1; Too few cells, row will be missing data

(MD056, table-column-count)